### PR TITLE
Update docs based on feedback in PR #11

### PR DESCRIPTION
**Pull request by hisaac**
Some updates based on @brentleyjones' comments in #11.

cc @regularberry

### CODE REVIEW

File: Docs/UPDATING.md [hunk 1]
```diff
@@ -52,8 +54,11 @@ To set up `BazelXCBBuildService` for debugging:
 	cp ~/Library/Developer/Xcode/DerivedData/$FOLDER/Build/Products/Debug/BazelXCBBuildService \
 	/Applications/Xcode-<main-version>.app/Contents/SharedFrameworks/XCBuild.framework/PlugIns/XCBBuildService.bundle/Contents/MacOS/
 	```
-1. In the "debugging" Xcode Go to "Debug" > "Attach to process by PID or Name" and type in "BazelXCBBuildService" if it's not already pre-filled.
+1. Prepare the debugger to attach to the service:
+	1. Open the scheme for this build, and navigate to the "Run" section on the left-hand sidebar
+	1. In the "Info" tab, for the "Launch" option, select "Wait for the executable to be launched"
 1. Set some breakpoints if desired
```
Comment (brentleyjones) [inline Docs/UPDATING.md:60]: I would make sure all of these are sentences too, with periods at the end.

**Review by brentleyjones — APPROVED** [review]

File: Docs/UPDATING.md (hunk 1 above)
Comment (hisaac) [reply Docs/UPDATING.md:60]: Whoops! Good catch.

File: Docs/UPDATING.md [hunk 2]
```diff
@@ -15,27 +15,38 @@ The `XCBBuildService` lives here: `$(XCODE_DIR)/Contents/SharedFrameworks/XCBuil
...
+* Restart Xcode.
+* Do some actions in Xcode, then view the logs. Are there any errors?
 * Make a change in the project fixing an error or adding extra logs you deem necessary.
-* Repeat
+* Repeat as necessary.
 
 ## Xcode Debugger
 
-To debug `BazelXCBBuildService` while it is being used by Xcode, you will need two versions of Xcode. (e.g. 12.5.0 and 13.0.0)
+To debug `BazelXCBBuildService` while it is being used by Xcode, you will need to have two versions of Xcode running.
```
Comment (brentleyjones) [inline Docs/UPDATING.md:37]:
```suggestion
To debug `BazelXCBBuildService` while it is being used by Xcode, you will need to have two instances of Xcode running:
```

File: Docs/UPDATING.md [hunk 3]
```diff
@@ -15,27 +15,38 @@ The `XCBBuildService` lives here: `$(XCODE_DIR)/Contents/SharedFrameworks/XCBuil
...
+
+1. "Main" Xcode: An instance of Xcode that uses our XCBBuildServiceProxy. This version will be used to build and run your app.
+1. "Debugging" Xcode: An instance of Xcode that uses the original XCBBuildService shipped with Xcode. This version will be used to debug the "main" instance of Xcode.
```
Comment (brentleyjones) [inline Docs/UPDATING.md:40]:
```suggestion
- "Main" Xcode: An instance of Xcode that uses our XCBBuildServiceProxy. This version will be used to build and run your app.
- "Debugging" Xcode: An instance of Xcode that uses the original XCBBuildService shipped with Xcode. This version will be used to debug the "main" instance of Xcode.
```

**Pull request merged**